### PR TITLE
fix(DB/Resistances): Incorrect Holy resistances

### LINKED ISSUE #5453: Remove ALL holy resistances
## CURRENT BEHAVIOUR:

About 140 mobs do have holy resistance 
image

image

## EXPECTED BLIZZLIKE BEHAVIOUR:

Just like the players, no creature should have holy resistance, it's what makes the holy school special, it can't be resisted.

## SOURCE:

Further proof : Blizz doesn't even have holy resist in their editing tool frm 2009 
image

## STEPS TO REPRODUCE THE PROBLEM:

1. 
2. 
3. 

## AC HASH/COMMIT:

d11d021f38e0e0db27e7c74da801402e7e5f17d9

## CUSTOMIZATIONS:

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/98101615-remove-all-holy-resistances?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by ErnestoGonzalezC**
## Changes Proposed:
-  Remove incorrect Holy resistances.

## Issues Addressed:
- Closes #5453

## Tests Performed:

- Verify that in the table `Creature_template_resistance` there are no creatures with resistances from school ``1``

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1619539350697557200.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1619539350697557200');
+
+DELETE FROM `creature_template_resistance` WHERE `School`=1 AND `CreatureID` IN (157,329,521,764,765,766,1501,1502,1812,1813,2359,2745,2761,2776,2794,2887,2919,3516,
+3568,3630,3631,3950,4028,4029,4030,4034,4423,4535,4661,5056,5291,
+5317,5320,5349,5356,5481,5485,5756,5761,5806,5834,5853,5881,5890,6074,6129,6220,6239,6492,6498,6499,6500,6550,6748,7031,7032,7104,7132,7226,7266,
+7768,7780,8211,8213,8281,8303,8400,8441,8667,8905,8906,8911,8981,8982,9017,9025,9026,9377,
+9397,9453,9598,10077,10442,10641,10882,10955,11357,11440,11441,11442,11443,11444,11480,11483,11484,11489,11491,11576,11577,11578,
+11669,11744,11745,11783,11784,11862,11871,13278,13279,13282,13285,13322,13456,13656,13696,13736,14025,14232,14234,14362,14399,14400,
+14454,14458,14460,14461,14464,15339,17359);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1619539350697557200.sql:9]: missing new line at the end of file 
image

Comment (ErnestoGonzalezC) [reply data/sql/updates/pending_db_world/rev_1619539350697557200.sql]: @FrancescoBorzi Did you mean the semicolon?

**Comment by Eldarix** [comment]
Checked, not all all them but a lot, and seems correctly updated with no holy resistances so far:

WoWScrnShot_052821_215313
WoWScrnShot_052821_215415
WoWScrnShot_052821_215623
WoWScrnShot_052821_215802
WoWScrnShot_052821_220141

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
thanks @Eldarix 

setting this as To Be Merged unless someone else has different feedback

cc @T1ti

**Pull request merged**